rdkPlugins/OOMCrash/README.md

```diff
@@ -0,0 +1,30 @@
+# Dobby RDK OOMCrash Plugin
+
+## Quick Start
+The main task of OOMCrash Plugin is to create a OOM crash file on the configured path when a container is crashed due to Out of Memory.
```

Comment:
You should now mention how the file is deleted + what is the naming convention of the OOM file.